services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaController.kt

```diff
@@ -111,7 +115,7 @@ class HankeKayttajaController(
     )
     @PreAuthorize("@hankeKayttajaAuthorizer.authorizeKayttajaId(#kayttajaId, 'VIEW')")
     fun getHankeKayttaja(@PathVariable kayttajaId: UUID): HankeKayttajaDto =
-        hankeKayttajaService.getKayttaja(kayttajaId).also {
+        hankeKayttajaService.getKayttaja(kayttajaId).toDto().also {
```

Comment:
This the kind of functionality I thought to implement with HakemusService.update: the service could return the full Hakemus domain object and then the Controller would create the dto with only the data needed. Should I refactor my solution into this kind of implementation? It would feel better for me, I think - but then I would need to again have Hakemus which I just deleted yesterday from my branch...

Reply:
You can refactor it in the next PR, which would add the controller?